.mvn/wrapper/maven-wrapper.properties

```
@@ -0,0 +1,2 @@
+distributionUrl=https://repo.maven.apache.org/maven2/org/apache/maven/apache-maven/3.5.4/apache-maven-3.5.4-bin.zip
```

we use 3.6.3 on our infrastructure currently

```suggestion
distributionUrl=https://repo.maven.apache.org/maven2/org/apache/maven/apache-maven/3.6.3/apache-maven-3.6.3-bin.zip
```

---

The docs recommended 3.5.4 or above.
@oleg-nenashev This is an example of the kind of problem a wrapper solves.